rclpy/rclpy/node.py

```diff
@@ -470,7 +470,7 @@ def declare_parameters(
                 descriptor.dynamic_typing = True
 
             if isinstance(second_arg, Parameter.Type):
-                if second_arg.value == Parameter.Type.NOT_SET:
+                if second_arg == Parameter.Type.NOT_SET:
```

Comment:
Is it necessary to update https://github.com/ros2/rclpy/blob/594669deb91451a7569e4f859cbc5d20ce93c200/rclpy/rclpy/node.py#L480?

Reply:
The type of `descriptor.type` is `'type': 'uint8',`, so getting the value from an enum is correct.

Reply:
yes